Update users.md

Jarli01 opened the pull request:
Updated and included a permissions matrix, rather than the list permissions.

### Description

_Short explanation of this PR (feel free to re-use commit message)_

### Checklist

- Commit
  - Title follows [commit conventions](https://bit.ly/commit-conventions)
  - Reference the relevant issue (`Fixes #007`, `See xoa-support#42`, `See https://...`)
  - If bug fix, add `Introduced by`
- Changelog
  - If visible by XOA users, add changelog entry
  - Update "Packages to release" in `CHANGELOG.unreleased.md`
- PR
  - If UI changes, add screenshots
  - If not finished or not tested, open as _Draft_

Jarli01 committed:
Update users.md

Updated and included a permissions matrix, rather than the list permissions.

olivierlambert reviewed (changes requested):
If it's possible, I would prefer a Markdown table, because it means anyone could maintain it without modifying a picture. I know it's a bit more work, but if you need help, let me know 👍

Jarli01 commented:
Not a problem, let me convert and resub.

Jarli01 committed:
Update users.md

Converted to Markdown, I did lose my header columns though which delineate which permissions apply to a VM or a Host/Pool

Jarli01 committed:
Merge branch 'master' into patch-2

Jarli01 committed:
Merge branch 'master' into patch-2

Jarli01 committed:
Merge branch 'master' into patch-2

Jarli01 reviewed (commented):
Converted to markdown

Jarli01 reviewed (commented):
adjusted per request - can this be reviewed?

Jarli01 committed:
Merge branch 'master' into patch-2

Jarli01 commented:
Here is a vertical view version which may work better on the website. 

| |Administrator|Operator|Viewer|
|:----|:----|:----|:----|
|Create|x| |
|Delete|x| |
|Migrate|x| |
|Modify – CPU, RAM, DISK etc|x| |
|Clone|x| |
|Copy|x| |
|Convert to Template|x| |
|Snapshot|x| |
|Export|x| |
|Attach/Remove Disk (VDIs)|x| |
|Insert/Eject Disk (ISO)|x|x| |
|General – Overview|x|x|x|
|Stats – Overview|x|x|x|
|Console – Remote Access|x|x| |
|Console – Start, Shutdown, Reboot|x|x| |
|Network Management – Conn/Disc.|x|| |
|Network Management – Delete|x| |
|Disk Management – Overview|x|x| |
|Disk Management – Remove from VDI|x| |
|Disk Management – Delete from VM|x| |
|Snapshot – Overview|x|x| |
|Snapshot – Create/Delete|x|| |
|Logs|x|x| |
|Logs – Delete|x| |
|Advanced – Overview|x|x| |
|Recovery Start|x|| |
|Start on…|x| |
|Clone|x| |
|Convert to Template|x| |
|Remove|x| |
|Boot Order – Change|x|| |
|Auto power on|x| |
|Protect from accidental deletion| |
|Protect from accidental shutdown|x| |
|Windows Update Tools|x| |
|HA|x| |
|GPUs|x| |
|NIC type|x| |
|VGA|x| |
|Video RAM|x| |
|Boot Firmware|x| |
